lsp-ui-doc.el

```diff
@@ -706,8 +750,13 @@ HEIGHT is the documentation number of lines."
 
 (defun lsp-ui-doc--make-request nil
   "Request the documentation to the LS."
+  (and (not track-mouse)
+       lsp-ui-doc-show-with-mouse
+       (setq-local track-mouse t))
   (when (and (not (eq this-command 'lsp-ui-doc-hide))
              (not (eq this-command 'keyboard-quit))
+             (not (eq this-command 'lsp-ui-doc--handle-mouse-movement))
+             (not (eq this-command 'ignore))
```

Comment:
I think you can use `memq` function here. This is a lot simpler.

Reply:
Yep I will change that, thanks